src/components/SignIn.js

```diff
@@ -117,15 +119,39 @@ export default function SignIn(props) {
     };
     return disabled ?
       Spinner :
-      h(GoogleLogin,
-        isNil(customStyle) ? defaultProps : {
-          render: (props) => button({
-            className: 'btn-primary',
-            onClick: props.onClick,
-            style: customStyle
-          }, 'Submit a Data Access Request'),
-          ...defaultProps
-        });
+      div({
+        style: {
+          display: 'flex'
+        }
+      }, [
+        h(GoogleLogin,
+          isNil(customStyle) ? defaultProps : {
+            render: (props) => button({
+              className: 'btn-primary',
+              onClick: props.onClick,
+              style: customStyle
+            }, 'Submit a Data Access Request'),
+            ...defaultProps
+          }),
+        a({
+          className: 'navbar-duos-icon-help',
+          style: {
+            color: 'white',
+            height: 16, width: 16,
+            marginLeft: 5
+          },
+          href: 'https://broad-duos.zendesk.com/hc/en-us/articles/6160103983771-How-to-Create-a-Google-Account-with-a-Non-Google-Email',
+          'data-for': 'tip_google-help',
+          'data-tip': 'No Google account? Click here!'
+        }),
+        h(ReactTooltip, {
+          id: 'tip_google-help',
+          place: 'top',
```

Comment:
I'm not really sure this "placement" doesnt seem to actually change anything. It says "top", but the tooltip ends up on the left... When I change it to "bottom", it still shows on the left! Anyway, I figure it's not that big of a deal. If you know what's wrong and how to update, please let me know (for academic curiosity's sake!)